interactions/api/http/http.py

```diff
@@ -0,0 +1,122 @@
+from typing import Any, Optional, Tuple
+
+import interactions.api.cache
+
+from ...api.cache import Cache
+from .channel import _ChannelRequest
+from .emoji import _EmojiRequest
+from .guild import _GuildRequest
+from .interaction import _InteractionRequest
+from .member import _MemberRequest
+from .message import _MessageRequest
+from .reaction import _ReactionRequest
+from .request import _Request
+from .route import Route
+from .scheduledEvent import _ScheduledEventRequest
+from .sticker import _StickerRequest
+from .thread import _ThreadRequest
+from .user import _UserRequest
+from .webhook import _WebhookRequest
+
+
+class HTTPClient(
```

Comment:
Let's rename the `http.py` file to `client.py` since we're housing only the `HTTPClient` and respective subclassing.